### PR TITLE
[zh] sync concepts/policy/node-resource-managers.md

### PR DESCRIPTION
**Pull request by jiecloud**
[zh] sync concepts/policy/node-resource-managers.md

### Issue

Closes: #

### CODE REVIEW

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 1]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+<!--
+- `none`: the `none` policy explicitly enables the existing default CPU
+affinity scheme, providing no affinity beyond what the OS scheduler does
+automatically.  Limits on CPU usage for
+[Guaranteed pods](/docs/concepts/workloads/pods/pod-qos/) and
+[Burstable pods](/docs/concepts/workloads/pods/pod-qos/)
+are enforced using CFS quota.
+-->
+- `none`：`none` 策略显式启用现有的默认 CPU 亲和性方案，除了操作系统调度器自动执行的操作外，不提供任何亲和性。
+    使用 CFS 配额强制执行 [Guaranteed Pod](/zh-cn/docs/concepts/workloads/pods/pod-qos/) 和 [Burstable Pod](/zh-cn/docs/concepts/workloads/pods/pod-qos/) 的 CPU 使用限制。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:84]:
```suggestion
  使用 CFS 配额对 [Guaranteed Pod](/zh-cn/docs/concepts/workloads/pods/pod-qos/) 和
  [Burstable Pod](/zh-cn/docs/concepts/workloads/pods/pod-qos/) 强制执行 CPU 用量限制。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 2]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+
+<!--
+- `static`: the `static` policy allows containers in `Guaranteed` pods with integer CPU
+`requests` access to exclusive CPUs on the node. This exclusivity is enforced
+using the [cpuset cgroup controller](https://www.kernel.org/doc/Documentation/cgroup-v2.txt).
+-->
+- `static`：`static` 策略允许具有整数 CPU `requests` 的 `Guaranteed` Pod 中的容器访问节点上的独占 CPU。
+    这种独占性是使用 [cpuset cgroup 控制器](https://www.kernel.org/doc/Documentation/cgroup-v2.txt) 强制执行的。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:92]:
```suggestion
  这种独占性是使用 [cpuset cgroup 控制器](https://www.kernel.org/doc/Documentation/cgroup-v2.txt)强制执行的。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 3]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+`distribute-cpus-across-cores`（alpha，默认隐藏）：
+: 在不同的物理核心之间分配虚拟核心（有时称为硬件线程）（自 Kubernetes v1.31 起可用）
+`distribute-cpus-across-numa`（alpha，默认隐藏）：
+: 在不同的 NUMA 域之间分配 CPU，力求在所选域之间实现均匀平衡（自 Kubernetes v1.23 起可用）
+`full-pcpus-only`（beta，默认可见）：
+: 始终分配完整的物理核心（自 Kubernetes v1.22 起可用）
+`strict-cpu-reservation`（alpha，默认隐藏）：
+: 阻止所有 Pod（无论其服务质量类别如何）在预留的 CPU 上运行（自 Kubernetes v1.32 起可用）
+`prefer-align-cpus-by-uncorecache`（alpha，默认隐藏）：
+: 尽可能通过非核心（最后一级）缓存边界对齐 CPU（自 Kubernetes v1.32 起可用）
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:287]:
```suggestion
`align-by-socket`（Alpha，默认隐藏）：
: 通过物理包/套接字边界（而不是逻辑 NUMA 边界）对齐 CPU（自 Kubernetes v1.25 起可用）

`distribute-cpus-across-cores`（Alpha，默认隐藏）：
: 在不同的物理核心之间分配虚拟核心（有时称为硬件线程）（自 Kubernetes v1.31 起可用）

`distribute-cpus-across-numa`（Alpha，默认隐藏）：
: 在不同的 NUMA 域之间分配 CPU，力求在所选域之间实现均匀平衡（自 Kubernetes v1.23 起可用）

`full-pcpus-only`（Beta，默认可见）：
: 始终分配完整的物理核心（自 Kubernetes v1.22 起可用）

`strict-cpu-reservation`（Alpha，默认隐藏）：
: 阻止所有 Pod（无论其服务质量类别如何）在预留的 CPU 上运行（自 Kubernetes v1.32 起可用）

`prefer-align-cpus-by-uncorecache`（Alpha，默认隐藏）：
: 尽可能通过非核心（最后一级）缓存边界对齐 CPU（自 Kubernetes v1.32 起可用）
```
这部分渲染有问题，需要加一些空行。参见预览：
https://deploy-preview-49170--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/policy/node-resource-managers/#cpu-policy-static--options

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 4]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+This can cause undesired bottlenecks in parallel code relying on barriers (and
+similar synchronization primitives), as this type of code tends to run only as
+fast as its slowest worker (which is slowed down by the fact that fewer CPUs
+are available on at least one NUMA node).
+By distributing CPUs evenly across NUMA nodes, application developers can more
+easily ensure that no single worker suffers from NUMA effects more than any
+other, improving the overall performance of these types of applications.
+-->
+如果指定了 distribute-cpus-across-numa 策略选项，则在需要多个 NUMA 节点来满足分配的情况下，static 策略将跨 NUMA 节点均匀分配 CPU。
+默认情况下，CPUManager 会将 CPU 打包到一个 NUMA 节点上，直到它被填满，任何剩余的 CPU 只会溢出到下一个 NUMA 节点。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:346]:
```suggestion
默认情况下，CPUManager 会将 CPU 打包到一个 NUMA 节点上，直到它被填满，剩余的所有 CPU 会溢出到下一个 NUMA 节点。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 5]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+这可能会导致依赖于障碍（和类似的同步原语）的并行代码出现不希望的瓶颈，因为这种类型的代码往往只会以其最慢的工作程序的速度运行（最慢的工作程序会因至少一个 NUMA 节点上的可用 CPU 较少而减速）。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:347]:
```suggestion
这可能会导致依赖于障碍（和类似的同步原语）的并行代码出现不希望的瓶颈，、因为这种类型的代码往往只会以其最慢的工作程序的速度运行（这一工作程序因为至少一个 NUMA 节点上的可用 CPU 较少而被减速）。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 6]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+通过在 NUMA 节点之间均匀分配 CPU，应用程序开发人员可以更轻松地确保没有单个工作程序比任何其他工作程序遭受更多的 NUMA 影响，从而提高这些类型应用程序的整体性能。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:348]:
```suggestion
通过在跨多个 NUMA 节点均匀分配 CPU，应用程序开发人员可以更轻松地确保没有单个工作程序比所有其他工作程序受
NUMA 影响更严重，从而提高这些类型的应用的整体性能。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 7]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+pulled from more than one NUMA node to satisfy the allocation. Although it
+tries to ensure that all CPUs are allocated from the _minimum_ number of NUMA
+nodes, there is no guarantee that those NUMA nodes will be on the same socket.
+By directing the `CPUManager` to explicitly align CPUs at the socket boundary
+rather than the NUMA boundary, we are able to avoid such issues. Note, this
+policy option is not compatible with `TopologyManager` `single-numa-node`
+policy and does not apply to hardware where the number of sockets is greater
+than number of NUMA nodes.
+-->
+如果指定了 align-by-socket 策略选项，则在决定如何将 CPU 分配给容器时，CPU 将被视为在套接字边界处对齐。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:366]:
```suggestion
如果指定了 align-by-socket 策略选项，则在决定如何将 CPU 分配给容器时，CPU 将被视为以插槽为边界对齐。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 8]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+默认情况下，CPUManager 会在 NUMA 边界处对齐 CPU 分配，如果需要从多个 NUMA 节点提取 CPU 才能满足分配，则可能会导致性能下降。
+虽然它试图确保所有 CPU 都从_最少_数量的 NUMA 节点中分配，但无法保证这些 NUMA 节点会在同一套接字上。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:368]:
```suggestion
虽然它试图确保所有 CPU 都从_最少_数量的 NUMA 节点中分配，但无法保证这些 NUMA 节点会在同一插槽上。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 9]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+通过指示 CPUManager 在套接字边界而不是 NUMA 边界处显式对齐 CPU，我们可以避免此类问题。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:369]:
```suggestion
通过指示 CPUManager 以插槽为边界而不是以 NUMA 节点为边界显式对齐 CPU，我们可以避免此类问题。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 10]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+请注意，此策略选项与 TopologyManager single-numa-node 策略不兼容，并且不适用于套接字数量大于 NUMA 节点数量的硬件。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:370]:
```suggestion
请注意，此策略选项与 TopologyManager 的 `single-numa-node` 策略不兼容，
并且不适用于插槽数量大于 NUMA 节点数量的硬件。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 11]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+in performance bottlenecks. By enabling the `distribute-cpus-across-cores` policy,
+the static policy ensures that cpus are distributed across as many physical cores
+as possible, reducing the contention on the same physical core and thereby
+improving overall performance. However, it is important to note that this strategy
+might be less effective when the system is heavily loaded. Under such conditions,
+the benefit of reducing contention diminishes. Conversely, default behavior
+can help in reducing inter-core communication overhead, potentially providing
+better performance under high load conditions.
+-->
+如果指定了 distribute-cpus-across-cores 策略选项，则 static 策略将尝试在不同的物理核心之间分配虚拟核心（硬件线程）。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:389]:
```suggestion
如果指定了 `distribute-cpus-across-cores` 策略选项，则 static
策略将尝试跨多个不同的物理核来分配虚拟核（硬件线程）。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 12]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+默认情况下，CPUManager 倾向于将 CPU 打包到尽可能少的物理核心上，这可能会导致同一物理核心上的 CPU 之间发生争用，并导致性能瓶颈。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:390]:
```suggestion
默认情况下，CPUManager 倾向于将 CPU 打包到尽可能少的物理核上，这可能会导致同一物理核上的 CPU
之间发生争用，并导致性能瓶颈。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 13]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+通过启用 distribute-cpus-across-cores 策略，static 策略可确保 CPU 分布在尽可能多的物理核心上，从而减少同一物理核心上的争用，从而提高整体性能。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:391]:
```suggestion
通过启用 `distribute-cpus-across-cores` 策略，static 策略可确保 CPU 分布在尽可能多的物理核上，
从而减少同一物理核上的争用，从而提高整体性能。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 14]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+但是，重要的是要注意，当系统负载过重时，此策略的效果可能会降低。
+在这种情况下，减少争用的好处会减少。
+相反，默认行为可以帮助减少核心间的通信开销，从而可能在高负载条件下提供更好的性能。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:394]:
```suggestion
相反，默认行为可以帮助减少处理器核之间的通信开销，从而可能在高负载条件下提供更好的性能。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 15]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+and kubernetes system daemons.  More details of this parameter can be found on the
+[Explicitly Reserved CPU List](/docs/tasks/administer-cluster/reserve-compute-resources/#explicitly-reserved-cpu-list) page.
+By default this isolation is implemented only for guaranteed pods with integer CPU requests not for burstable and best-effort pods
+(and guaranteed pods with fractional CPU requests). Admission is only comparing the cpu requests against the allocatable cpus.
+Since the cpu limit is higher than the request, the default behaviour allows burstable and best-effort pods to use up the capacity
+of `reservedSystemCPUs` and cause host OS services to starve in real life deployments.
+If the `strict-cpu-reservation` policy option is enabled, the static policy will not allow
+any workload to use the CPU cores specified in `reservedSystemCPUs`.
+-->
+KubeletConfiguration 中的 reservedSystemCPUs 参数
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:410]:
```suggestion
KubeletConfiguration 中的 `reservedSystemCPUs` 参数
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 16]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+或已弃用的 kubelet 命令行选项 --reserved-cpus 为操作系统系统守护程序和 Kubernetes 系统守护程序定义了显式的 CPU 集。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:411]:
```suggestion
或已弃用的 kubelet 命令行选项 `--reserved-cpus` 定义显式的 CPU 集合，
用来运行操作系统系统守护进程和 Kubernetes 系统守护进程。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 17]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+有关此参数的更多详细信息，请参见显式预留 CPU 列表页面。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:412]:
```suggestion
有关此参数的更多详细信息，请参见[显式预留 CPU 列表](/zh-cn/docs/tasks/administer-cluster/reserve-compute-resources/#explicitly-reserved-cpu-list)页面。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 18]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+默认情况下，此隔离仅针对具有整数 CPU 请求的保证型 Pod 实现，而不适用于突发型和尽力而为型 Pod
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:413]:
```suggestion
默认情况下，此隔离仅针对 CPU 请求数量为整数的 Guaranteed 类的 Pod 实现，
而不适用于 Burstable 和 BestEffort 类的 Pod
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 19]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+（以及具有小数 CPU 请求的保证型 Pod）。准入仅将 CPU 请求与可分配的 CPU 进行比较。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:414]:
```suggestion
（以及 CPU 请求数量包含小数的 Guaranteed 类 Pod）。执行准入判断时，系统仅将 CPU 请求与可分配的 CPU 进行比较。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 20]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+由于 CPU 限制高于请求，因此默认行为允许突发型和尽力而为型 Pod 占用 reservedSystemCPUs 的容量，并导致主机 OS 服务在实际部署中资源不足。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:415]:
```suggestion
由于 CPU 限制数量高于请求数量，因此默认行为允许 Burstable 和 BestEffort 类的 Pod 占用
`reservedSystemCPUs` 所预留的容量，并在实际部署中导致主机 OS 服务资源不足。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 21]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+如果启用了 strict-cpu-reservation 策略选项，则 static 策略将不允许任何工作负载使用 reservedSystemCPUs 中指定的 CPU 核心。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:416]:
```suggestion
如果启用了 `strict-cpu-reservation` 策略选项，则 static 策略将不允许任何工作负载使用
`reservedSystemCPUs` 中指定的 CPU 核。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 22]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+of the uncore cache. Allocation is performed on a best-effort basis, aiming to 
+affine as many CPUs as possible within the same uncore cache. If the container's 
+CPU requirement exceeds the CPU capacity of a single uncore cache, the `CPUManager` 
+minimizes the number of uncore caches used in order to maintain optimal uncore 
+cache alignment. Specific workloads can benefit in performance from the reduction 
+of inter-cache latency and noisy neighbors at the cache level. If the `CPUManager` 
+cannot align optimally while the node has sufficient resources, the container will 
+still be admitted using the default packed behavior.
+-->
+如果指定了 prefer-align-cpus-by-uncorecache 策略，则 static 策略将为各个容器分配 CPU 资源，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:436]:
```suggestion
如果指定了 `prefer-align-cpus-by-uncorecache` 策略，则 static 策略为各个容器分配 CPU 资源时，
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 23]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+以便分配给容器的所有 CPU 共享同一个非核心缓存块（也称为最后一级缓存或 LLC）。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:437]:
```suggestion
会让分配给容器的所有 CPU 共享同一个非处理核缓存块（也称为最后一级缓存或 LLC）。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 24]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+默认情况下，CPUManager 会紧密打包 CPU 分配，这可能会导致将多个非核心缓存中的 CPU 分配给容器。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:438]:
```suggestion
默认情况下，CPUManager 会压缩打包 CPU 分配，这可能会导致分配给容器的 CPU 使用来自多个非核心的高速缓存块。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 25]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+此选项使 CPUManager 能够以最大化非核心缓存的有效利用率的方式分配 CPU。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:439]:
```suggestion
此选项使 CPUManager 能够在分配 CPU 时将非核心缓存的有效利用率最大化。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 26]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+分配是在尽力而为的基础上执行的，旨在使尽可能多的 CPU 位于同一非核心缓存中。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:440]:
```suggestion
分配是在尽力而为的，目的是使共享同一非核心高速缓存的 CPU 个数尽可能多。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 27]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+如果容器的 CPU 需求超过了单个非核心缓存的 CPU 容量，则 CPUManager 会尽量减少使用的非核心缓存数量，以保持最佳的非核心缓存对齐。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:441]:
```suggestion
如果容器的 CPU 需求超过了单个非核心缓存对应的 CPU 个数，则 CPUManager
会尽量减少所使用的非核高速缓存数量，以保持最佳的非核高速缓存对齐。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 28]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+特定的工作负载可以从减少缓存级别的缓存间延迟和嘈杂的邻居中受益。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:442]:
```suggestion
某些的工作负载可以从降低缓存级别的缓存间延迟，减少嘈杂邻居的影响中受益。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 29]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+-->
+## 内存管理策略
+
+{{< feature-state feature_gate_name="MemoryManager" >}}
+
+<!--
+The Kubernetes *Memory Manager* enables the feature of guaranteed memory (and hugepages)
+allocation for pods in the `Guaranteed` {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}.
+-->
+Kubernetes 内存管理器（Memory Manager） 为 Guaranteed {{< glossary_tooltip text="QoS 类" term_id="qos-class" >}} 中的 Pod 启用了有保证的内存（和巨页）分配功能。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:456]:
```suggestion
Kubernetes 内存管理器（Memory Manager） 为 Guaranteed
{{< glossary_tooltip text="QoS 类" term_id="qos-class" >}} 中的 Pod
启用有保证的内存（和巨页）分配能力。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 30]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+
+<!--
+The Memory Manager employs hint generation protocol to yield the most suitable NUMA affinity for a pod.
+The Memory Manager feeds the central manager (*Topology Manager*) with these affinity hints.
+Based on both the hints and Topology Manager policy, the pod is rejected or admitted to the node.
+-->
+内存管理器采用提示生成协议，为 Pod 生成最合适的 NUMA 亲和性。
+内存管理器将这些亲和性提示馈送到中央管理器（拓扑管理器（Topology Manager））。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:464]:
```suggestion
内存管理器将这些亲和性提示提交到中央管理器，即拓扑管理器（Topology Manager）。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 31]
```diff
@@ -25,21 +25,459 @@ Kubernetes 提供了一组资源管理器，用于支持延迟敏感的、高吞
...
+根据提示和拓扑管理器策略，Pod 将被拒绝或允许进入节点。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:465]:
```suggestion
取决于提示信息和拓扑管理器的策略，Pod 将被拒绝或允许进入节点。
```

File: content/zh-cn/docs/concepts/policy/node-resource-managers.md [hunk 32]
```diff
@@ -52,24 +52,27 @@ hardware (for example, sharing CPUs across multiple Pods) or allocate hardware b
...
+或者通过专门划分一些资源来分配硬件（例如，为 Pod 独占使用分配一个或多个 CPU）。
 
 <!-- 
 By default, the kubelet uses [CFS quota](https://en.wikipedia.org/wiki/Completely_Fair_Scheduler)
 to enforce pod CPU limits.  When the node runs many CPU-bound pods, the workload can move to different CPU cores depending on whether the pod is throttled and which CPU cores are available at scheduling time. Many workloads are not sensitive to this migration and thus
 work fine without any intervention.
 -->
 默认情况下，kubelet 使用 [CFS 配额](https://en.wikipedia.org/wiki/Completely_Fair_Scheduler)
-来强制执行 Pod 的 CPU 限制。当节点运行许多 CPU 密集型 Pod 时，工作负载可能会移动到不同的 CPU 内核，具体取决于Pod 是否受到限制以及调度时哪些 CPU 内核可用。许多工作负载对这种迁移不敏感，因此无需任何干预即可正常工作。
+来强制执行 Pod 的 CPU 限制。当节点运行许多 CPU 密集型 Pod 时，工作负载可能会移动到不同的 CPU 内核，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/policy/node-resource-managers.md:63]: CPU Core 直接翻译为“ CPU 核”，不要翻译为”内核“、”核心“，请彻查一下。

Comment (jiecloud) [reply content/zh-cn/docs/concepts/policy/node-resource-managers.md]: @tengqm 谢谢指出及前面多处指导建议 ，cpu core 仅63，64行 2处，已修改，请审核。

**Comment by tengqm** [comment]
Please 1) mark the feedbacks as "resolved" if you agreed to the suggestion and made the changes accordingly; 2) squash your PR into a single commit.

/lgtm

**Comment by jiecloud** [comment]
> Please 1) mark the feedbacks as "resolved" if you agreed to the suggestion and made the changes accordingly; 2) squash your PR into a single commit.
> 
> /lgtm

已修改。

**Pull request merged**